addon.gradle

```diff
@@ -5,7 +5,7 @@ sourceSets.main {
 task initAVRcore(type: Exec) {
     description = "Updates (and inits) git submodules"
     group = "Build Setup"
-    commandLine("cmd", "git", "--git-dir=\"" + projectDir.toString() + "/.git\"", "submodule", "update", "--init", "--recursive", "--remote")
+    commandLine 'git', '--git-dir="' + projectDir.toString() + '\\.git"', 'submodule', 'update', '--init', '--recursive', '--remote'
```

Comment:
This seems ... suspect. Is there no better way to call git than shelling out with a patched together cmdline?

Reply:
I hope to find a way. Jitpack won't accept it. Any ideas?